pillowtop/tests/test_checkpoints.py

```diff
@@ -23,14 +28,6 @@ def test_get_or_create_empty(self):
         self.assertEqual('0', checkpoint['seq'])
         self.assertTrue(bool(checkpoint['timestamp']))
 
-
-class PillowCheckpointTest(SimpleTestCase):
```

Comment:
This is unrelated to the fix, but when I was searching for a place to add the test I found that two classes were named the same and thus the first one wasn't being run